### PR TITLE
20.5.2 rn

### PR DESCRIPTION
**Pull request by jochman**
- [x] Ready
20.5.2 rn

### CODE REVIEW

File: content-repo/extra-docs/releases/20.5.2.md [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+#### 5 New Integrations
+- __IllusiveNetworks__  
+The Illusive Attack Management API allows customers to retrieve detected incidents with a forensics timeline, attack surface insights, collect forensics on-demand, and manage a variety of operations with regard to deceptive entities, deception policies, and more.
+- __Bastille Networks__  
+RF monitoring for wireless intrusion detection and policy enforcement. Visit https://www.bastille.net for details.
+- __Logz.io__  
+Fetch & remediate security incidents identified by Logz.io Cloud SIEM.
+- __Digital Guardian__  
+Use Digital Guardian Integration to fetch incidents and to programmatically add or remove entries from watchlists and component lists.
+- __ThreatMiner__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:21]: its not a new one, was just converted to directory package format
take rn from https://github.com/demisto/content/blob/master/Packs/ThreatMiner/ReleaseNotes/1_0_1.md

File: content-repo/extra-docs/releases/20.5.2.md [hunk 2]
```diff
@@ -0,0 +1,271 @@
+## Cortex XSOAR Release Notes for version 20.5.2 (54345)
+##### Published on 26 May 2020
+
+### Breaking Changes
+- __TrueSTAR__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:5]:
```suggestion
- __TruSTAR__  
```

File: content-repo/extra-docs/releases/20.5.2.md [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+  - *Breaking changes:*
+    - *normalized_triage_score* argument replaced by *priority_event_score* in *trustar-get-phishing-submissions* and *trustar-get-phishing-indicators* command.
+    - Updated context outputs on *trustar-get-phishing-submissions* and *trustar-get-phishing-indicators*.  
+    - *normalized_source_score* argument replaced by *normalized_indicator_score* in *trustar-get-phishing-indicators* command.
+  - *Non Breaking changes:*
+    - Fixed *from_time* description on *trustar-get-phishing-indicators* and *trustar-get-phishing-submissions* to '24 hours ago'.
+    - Added -1 to list of default values in *priority_event_score* on *trustar-get-phishing-submissions*.
+    - Added -1 to list of default values in *priority_event_score* and *normalized_indicator_score* on *trustar-get-phishing-indicators*.
+- __Microsoft Graph Security__  
+Added **fetch-incidents** functionality.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:35]: lets be consistent with the fetch incidents, here it says `incidents` and formatted in bold while 2 lines below it says `incident` and in italic

File: content-repo/extra-docs/releases/20.5.2.md [hunk 4]
```diff
@@ -0,0 +1,271 @@
...
+Added the *Domain.Whois.QueryResult* output, which tells whether the query found a matching result.
+- __Recorded Future__  
+Fixed an issue in **recorded-future-get-related-entities** where the command output was mishandled.
+- __Kafka v2__  
+Added support for lz4 compressed messages.
+- __Palo Alto Networks PAN-OS__  
+  - Added the option to list predefined applications in PAN-OS 9.X in the ***panorama-get-applications*** command using the argument *predefined*.
+  - Fixed an issue where listing custom applications in PAN-OS 9.X using the ***panorama-get-applications***  command did not work properly.
+  - Fixed an issue where running the ***panorama-get-url-category*** command multiple times, displayed previous results in the war room.
+  - Replaced the spaces in the URL context output of the **!panorama-create-edl** command to *%20*.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:54]:
```suggestion
  - Replaced the spaces in the URL context output of the **panorama-create-edl** command to *%20*.
```

File: content-repo/extra-docs/releases/20.5.2.md [hunk 5]
```diff
@@ -0,0 +1,271 @@
...
+Fixed an issue when pulling an object that contains a date.
+- __Azure Compute v2__  
+Added support to authenticate using a self-deployed Azure application.
+- __Palo Alto Networks BPA__  
+  - Added an argument which enables you to download a Panorama report.
+  - Fixed an issue where proxy settings were not handled properly.
+
+#### Deprecated Integration
+- __ServiceNow__  
+Use the *ServiceNow v2* integration instead (available from 5.0.0).
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:99]: lets be consistent with the "from version" format
here we say `available from 5.0.0` and a few lines to the bottom we say `Available from Cortex XSOAR 5.5`

File: content-repo/extra-docs/releases/20.5.2.md [hunk 6]
```diff
@@ -0,0 +1,271 @@
...
+---
+### Scripts
+
+####  2 New Scripts
+- __BetweenDates__  
+Whether value is within a date range.
+- __BetweenHours__  
+Checks whether the given value is within the specified time (hour) range.
+
+#### 26 Improved Scripts
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:124]: there arent 26 scripts listed here

File: content-repo/extra-docs/releases/20.5.2.md [hunk 7]
```diff
@@ -0,0 +1,271 @@
...
+- __ElasticSearchDisplay__  
+Use the *Elasticsearch v2* integration instead.
+- __AwsGetInstanceInfo__  
+There is no replacement script.
+
+---
+### Playbooks
+
+####  10 New Playbooks
+- __Logz.Io Handle Alert__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:164]:
```suggestion
- __Logz.io Handle Alert__  
```

File: content-repo/extra-docs/releases/20.5.2.md [hunk 8]
```diff
@@ -0,0 +1,271 @@
...
+The playbook will not stop if Rasterize fails. This improves the playbook stability when rasterizing URLs of websites that are currently down.
+- __Phishing Investigation - Generic v2__  
+  - The playbook now uses Block Indicators - Generic v2 to block malicious indicators (off by default).
+  - Replaced the deprecated SendEmail automation with the send-mail command.
+
+---
+### Incident Fields
+Added incident fields for:
+- __Digital Guardian__  
+- __Logs.Io__
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:232]:
```suggestion
- __Logz.io__  
```

File: content-repo/extra-docs/releases/20.5.2.md [hunk 9]
```diff
@@ -0,0 +1,271 @@
...
+####  3 New Classification & Mapping
+- __Logz.io__  
+- __Digital Guardian__  
+- __IllusiveNetworks__  
+
+---
+### Reputations
+  - The **CVE CVSS Score** indicator type is only available in Demisto version 4.5.x and earlier.
+  - Fixed an issue where the "::" string was extracted as an IPv6 address.
+  - Added the **expiration** field, which defines when indicators of this type expire (in minutes). The default value is 10,080 (7 days). Available from Demisto v5.5.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:268]: same comment here about the from version

File: content-repo/extra-docs/releases/20.5.2.md [hunk 10]
```diff
@@ -0,0 +1,271 @@
+## Cortex XSOAR Release Notes for version 20.5.2 (54345)
+##### Published on 26 May 2020
+
+### Breaking Changes
+- __TrueSTAR__  
+Changed several arguments and outputs in **trustar-get-phishing-submissions** and **trustar-get-phishing-indicators** commands. See the **Integrations** section for full details.
```
Comment (Itay4) [inline content-repo/extra-docs/releases/20.5.2.md:6]: lets make the reference to the integrations clickable by adding a link to the integration section

File: content-repo/extra-docs/releases/20.5.2.md [hunk 11]
```diff
@@ -0,0 +1,269 @@
...
+---
+### Classification & Mapping
+
+####  3 New Classification & Mapping
+- __Logz.io__  
+- __Digital Guardian__  
+- __IllusiveNetworks__  
+
+---
+### Reputations
```
Comment (glicht) [inline content-repo/extra-docs/releases/20.5.2.md:263]: Not clear what this section is. It looks like it contains garbage.

**Comment by glicht** [comment]
@jochman I did a small commit. The title should be with one `#`.

File: content-repo/extra-docs/releases/20.5.2.md (hunk 11 above)
Comment (jochman) [reply content-repo/extra-docs/releases/20.5.2.md:263]: I think we can remove this whole section. @kirbles19 What do you think?

Comment (kirbles19) [reply content-repo/extra-docs/releases/20.5.2.md:263]: Which, Reputations?

Comment (jochman) [reply content-repo/extra-docs/releases/20.5.2.md:263]: yeap

Comment (kirbles19) [reply content-repo/extra-docs/releases/20.5.2.md:263]: I'll take it out of my version

**Comment by Itay4** [comment]
@glicht we are done here, want to have a final review?

**Pull request merged**